src/window.cpp

```diff
@@ -98,10 +101,10 @@ window_impl::window_impl(int pWidth, int pHeight, const char* pTitle,
         }
     }
 
-    // Set context (before glewInit())
+    /* Set context (before glewInit()) */
     MakeContextCurrent(this);
 
-    //GLEW Initialization - Must be done
+    /* GLEW Initialization - Must be done */
```

Comment:
Using commenting style (`/**/`) most of the places, so making it consistent across all files.